lib/core/composables/mixpanel.ts

```diff
@@ -84,6 +84,8 @@ export function useMixpanel() {
         distinct_id: hashedEmail,
         // eslint-disable-next-line camelcase
         server_id: hashedServer,
+        // eslint-disable-next-line camelcase
+        server_domain: serverHostname,
```

Comment:
@iainsproat i dont know is there a practical difference here by having hashed and unhashed version.

also, if it is needed, is `server_domain` correct prop?